### PR TITLE
A solution for displaying a series with two colours #92

### PR DESCRIPTION
**Pull request by jacques-lebourgeois**
Here's a solution that doesn't change the library but requires two series to be produced depending on whether or not the objective is achieved

### CODE REVIEW

**Review by louismaximepiton — APPROVED** [review]
I'm fine with this use case

**Comment by julien-deramond** [comment]
https://github.com/Orange-OpenSource/ods-charts/pull/168 is merged so this PR should be updated to use title bars probably. Please check the other PRs too for some updates if there are some new pages ;)

**Comment by jacques-lebourgeois** [comment]
A complement : we probably should use colors from libs and not custom colors in this example

File: docs/use_cases/two-colors-serie.html [hunk 1]
```diff
@@ -0,0 +1,245 @@
...
+        ///////////////////////////////////////////////////
+        // ODSCharts
+        ///////////////////////////////////////////////////
+        // Build the theme
+        var themeManager = ODSCharts.getThemeManager({
+          // Possibility to specify a Color out of ODS recommandations by given its value
+          // Cf https://ods-charts.netlify.app/api/interfaces/odschartsthemeoptions
+          // https://ods-charts.netlify.app/api/types/odschartscustomcategoricalcolor
+          // It can be the string value of the color
+          categoricalColors: ['#32C832', '#CD3C14', '#ff7900'],
```
Comment (julien-deramond) [inline docs/use_cases/two-colors-serie.html:208]: When you mentioned colours in https://github.com/Orange-OpenSource/ods-charts/pull/161#issuecomment-2009593856, you were referring to this line? That we should rather use the default colours from the library instead? If so, I definitely agree :)

**Comment by julien-deramond** [comment]
https://github.com/Orange-OpenSource/ods-charts/pull/158 has been merged, please apply the same modifications here.

File: docs/use_cases/two-colors-serie.html (hunk 1 above)
Comment (julien-deramond) [reply docs/use_cases/two-colors-serie.html]: Dropped the custom colours in https://github.com/Orange-OpenSource/ods-charts/pull/161/commits/a8e8bdc29750e949b8ce1a3deb351169659bc3ee

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**